src/marshmallow/decorators.py

```diff
@@ -77,12 +77,12 @@ class MarshmallowHook:
     )  # type: Optional[Dict[Union[Tuple[str, bool], str], Any]]
 
 
-def validates(field_name: str) -> Callable[..., Any]:
+def validates(*field_names: str) -> Callable[..., Any]:
     """Register a field validator.
 
     :param str field_name: Name of the field that the method validates.
     """
-    return set_hook(None, VALIDATES, field_name=field_name)
+    return set_hook(None, VALIDATES, field_names=field_names)
```

Comment:
Should we try to preserve this hook interface for a minor release? The docstring for `set_hook` says "You should not need to use this method directly". Any objection to making `validates` return a wrapper that loops the calls to `set_hook` instead?

Reply:
I don't see how the wrapper will work if we loop and call set_hook, `__marshmallow_hook__` attr gets overwritten each time.

Reply:
https://github.com/marshmallow-code/marshmallow/commit/c7bda74799ffd82640505f64b8b0de33213a84e8

Reply:
It won't work, try this. It will only register the last field given in the decorator

```
from marshmallow import (
    Schema,
    fields,
    pre_dump,
    post_dump,
    pre_load,
    post_load,
    # pre_validate,
    validates,
    validates_schema,
    ValidationError,
    EXCLUDE,
    INCLUDE,
    RAISE,
)

class TestValidates(Schema):
    value = fields.Str()
    another_value = fields.Str()
    
    @validates('value', 'another_value')
    def validates_function(self, data, **kwargs):
        print(f'Inside validates: {{ {data}:{data} }}')


schema = TestValidates()
print(schema._hooks['validates'])
validator = getattr(schema, schema._hooks['validates'][0])
print(validator.__marshmallow_hook__)
print()



schema.load({
    'value': 'value',
    'another_value': 'another_value'
})
```